include/openssl/tls1.h

```diff
@@ -232,6 +232,21 @@ __owur int SSL_export_keying_material(SSL *s, unsigned char *out, size_t olen,
                                       const unsigned char *context,
                                       size_t contextlen, int use_context);
 
+/*
+ * SSL_export_keying_material_early exports a value derived from the
+ * early exporter master secret, as specified in
+ * https://tools.ietf.org/html/draft-ietf-tls-tls13-23. It writes
+ * |olen| bytes to |out| given a label and optional context. (Since a
+ * zero length context is allowed, the |use_context| flag controls
+ * whether a context is included.) It returns 1 on success and 0 or -1
+ * otherwise.
+ */
+__owur int SSL_export_keying_material_early(SSL *s, unsigned char *out,
```

Comment:
There should be documentation for this function and a test (most likely in `test_sslapi.c`).

Reply:
Are we only allowed to call this in cases where 0-RTT is accepted? The spec isn't 100% clear to me on this:

>  The early_exporter_master_secret is defined for use in
>    settings where an exporter is needed for 0-RTT data.

Does this imply 0-RTT is accepted?

Reply:
Not sure about the spec, but in practice, we probably don't use early secret if 0-RTT is not accepted.

Will check test code, and documentation.  Do you think that the manual page of this function should be written in the same page of SSL_export_keying_material?

Reply:
> Do you think that the manual page of this function should be written in the same page of SSL_export_keying_material?

Yes that makes sense